src/permitInfo/utils/getTokensFromTokenList.ts

```diff
@@ -1,19 +1,18 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
 import { readFileSync } from 'node:fs'
+import { join } from 'node:path'
 import { BASE_PATH } from '../const'
 import { Token } from '../types'
-import { join } from 'node:path'
-import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
 const tokenListsByNetwork: Record<SupportedChainId, string> = {
   [SupportedChainId.MAINNET]: 'CowSwap.json',
+  [SupportedChainId.ARBITRUM_ONE]: 'CowSwap.json',
```

Comment:
At some point, we might want to break by network the lists, right?

Reply:
Not sure. I'm inclined to think now that we should keep them all in the same file, except test chains.